mindjunkies/courses/forms.py

```
@@ -1,36 +1,50 @@
 from django import forms
 from django.forms import inlineformset_factory
+from taggit.forms import TagWidget
 
 from .models import Course, CourseInfo, CourseToken, Rating
```

Ruff (0.8.2)

`.models.CourseToken` imported but unused

Remove unused import: `.models.CourseToken`

(F401)